net/acme/files/run.sh

```diff
@@ -335,7 +335,15 @@ issue_cert()
 		# commit and reload is in post_checks
 	fi
 
-	if [ -e /etc/init.d/nginx ] && [ "$update_nginx" -eq "1" ]; then
+	if command -v nginx-util 2>/dev/null && [ "$update_nginx" -eq "1" ]; then
+		for domain in $domains; do
+			nginx-util add_ssl "${domain}" acme "${domain_dir}/fullchain.cer" \
+				"${domain_dir}/${main_domain}.key"
+		done
+		# reload is in post_checks
+	fi
+
+	if [ -w /etc/nginx/nginx.conf ] && [ "$update_nginx" -eq "1" ]; then
```

Comment:
Hmm, this will potentially end up doing both updates; is that a problem? Will nginx-util always succeed, or can it tell if it's not being used? If so, maybe it's better to fall back and only do the config file update if nginx-util is not installed or if it can't update things?

Reply:
Although it should not be a problem, the nginx-util only returns 0 if successfully; I changed the code to try editing the `nginx.conf` only when `nginx-util` fails or is not there at all ...